src/components/common/Tabs/TabItem.tsx

```diff
@@ -3,20 +3,13 @@ import styled from 'styled-components'
 
 interface TabProps {
   title: string
-  readonly id: number
-  onTabClick: (arg: number) => void
-  isActive: boolean
-  readonly activeColor?: string
+  onTabClick: () => void
+  readonly activeColor: string | false
 }
 
-interface WrapperProps {
-  readonly activeColor?: string
-  isActive: boolean
-}
-
-const Wrapper = styled.li<WrapperProps>`
-  background: ${({ isActive, activeColor }): string => `var(${isActive ? activeColor : '--color-primary'})`};
-  color: ${({ isActive }): string => `var(--color-text-${isActive ? 'primary' : 'secondary2'})`};
+const Wrapper = styled.li<Pick<TabProps, 'activeColor'>>`
+  background: ${({ activeColor }): string => `var(${activeColor || '--color-primary'})`};
```

Comment:
Here we destructure `activeColor` from `props` correct?

Do we need the $ in front because of it being standard interpolated template literals?

Reply:
> Do we need the $ in front because of it being standard interpolated template literals?
do you mean in the variable name? like `$activeColor`? Normally yes we would name any prop that is passed into a `styled-component` this way but with the way we were passing it I didnt change it. Maybe not necessary for this project.

Reply:
> Here we destructure `activeColor` from `props` correct?
> 
> Do we need the $ in front because of it being standard interpolated template literals?

also dropped `isActive` entirely

Reply:
So in this case if we were to use $activeColor vs. activeColor (as the destructured variable from `props`) that is the same, just a specific naming convention (prepending the $ sign) to make it distinctive for a reason?